qemu/tests/win_virtio_driver_installer_repair.py

```diff
@@ -19,106 +17,72 @@ def run(test, params, env):
     1) Create shared directories on the host.
     2) Run virtiofsd daemons on the host.
     3) Boot guest with all virtio device.
-    4) Install driver via virtio-win-guest-tools.exe.
-    5) Run virtio-win-guest-tools.exe signature check command in guest.
-    6) Verify the qemu-ga version match expected version.
-    7) Run driver signature check command in guest.
+    4) Run driver signature check command in guest.
        Verify target driver.
-    8) Run virtio-win-guest-tools.exe repair test by uninstall
-       the target driver.
-    9) Run the driver function test after virtio-win-guest-tools.exe repair.
-    10) Repeat step 9 and 10 for other drivers.
+    5) Uninstall dirvers by msi uninstall.
+    6) Run virtio-win-guest-tools.exe repair test.
+    7) Run the driver function test after repair.
 
     :param test: QEMU test object
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment
     """
-    devcon_path = params["devcon_path"]
     run_install_cmd = params["run_install_cmd"]
-    installer_pkg_check_cmd = params["installer_pkg_check_cmd"]
-
-    # gagent version check test config
-    qemu_ga_pkg = params["qemu_ga_pkg"]
-    gagent_pkg_info_cmd = params["gagent_pkg_info_cmd"]
-    gagent_install_cmd = params["gagent_install_cmd"]
-    gagent_uninstall_cmd = params["gagent_uninstall_cmd"]
+    run_repair_cmd = params["run_repair_cmd"]
 
     vm = env.get_vm(params["main_vm"])
     session = vm.wait_for_login()
 
-    expected_gagent_version = win_driver_installer_test.install_gagent(
-        session, test,
-        qemu_ga_pkg,
-        gagent_install_cmd,
-        gagent_pkg_info_cmd)
-    win_driver_installer_test.uninstall_gagent(session, test,
-                                               gagent_uninstall_cmd)
```

Comment:
@xiagao Could you elaborate on the reason to remove gagent re-install testing? @6-dehan Could you check if we have other tests covering this scenario or even does it need to be covered? Thanks.

Reply:
@vivianQizhu Sure,these codes are just to get qemu-guest-agent version and check it, and there are repeated steps in [win_](https://github.com/autotest/tp-qemu/blob/master/qemu/tests/win_virtio_driver_install_by_installer.py) and https://github.com/autotest/tp-qemu/blob/master/qemu/tests/win_virtio_driver_installer_uninstall.py

Reply:
Thanks @xiagao . @6-dehan Request your ACK on this change, thanks.

Reply:
there are also other two scripts contain this scenario: 
`win_virtio_driver_install_by_installer.py:1`
`win_virtio_driver_install_by_installer.py:1`

It does not affect the agent because other scripts cover it. 
ACK